package_info.json

```diff
@@ -1,4 +1,4 @@
 {
   "package_name": "th2-common",
-  "package_version": "2.2.10"
+  "package_version": "2.2.11"
```

Comment:
Please, change version according to latest changes

Reply:
Ok, but why do we not increase the version?

Reply:
Not the same, 2.3.1, actually